content/en/platform/corda/5.0/deploying-operating/external-messaging/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+date: '2023-06-28'
+title: "External Messaging Administration"
+project: corda
+version: 'Corda 5.0 Beta 4'
+menu:
+  corda5:
+    identifier: corda5-cluster-external-messaging
+    parent: corda5-cluster
+    weight: 10000
+section_menu: corda5
+---
+
+# External Messaging Administration
+
+CorDapp flows can send simple messages via Kafka to external systems. For more information, see [External Messaging CorDapps]({{< relref "../../developing-applications/external-messaging.md" >}}). This section describes the cluster adminstration tasks required for CorDapps that implement external messaging. It contains the following:
+* [Configuring External Messaging Routes](#configuring-external-messaging-routes)
+* [Creating Kafka Topics](#creating-kafka-topics)
+
+## Configuring External Messaging Routes
+
+The <a href="./config/fields/externalmessaging.html">`externalMessaging` configuration section</a> specifies how Corda creates the required messaging routes when [creating a new virtual node]({{< relref "../../application-networks/creating/members/cpi.md">}}). 
+You can update these using the <a href="../../reference/rest-api/C5_OpenAPI.html#tag/Configuration-API/operation/put_config">`config` endpoint</a> of the REST API. For example:
```

Comment:
The hyperlink does not seem to be working. In the preview, I get the following error:
![image](https://github.com/corda/corda-docs-portal/assets/6329837/f5bfbfc9-f835-4ed4-922c-00066871d69e)